client/posts/menu.ts

```diff
@@ -239,6 +240,7 @@ async function getSameIPPosts(m: Post): Promise<PostData[]> {
 		return
 	}
 
+	send(message.meidoVision, [m.id, m.op])
```

Comment:
So that other clients are updated whenever meidovision is used, for the post status.

Reply:
Just send it through webscokets with feeds. Anything unclear you have questions about?

Reply:
Isn't that what I'm doing here?
If there's something different I need to do, give me an example please.
Also, whatever I do, I do it in that function it currently is in, right?

Reply:
>If there's something different I need to do, give me an example please.

If you are doing everything properly through websockets, why do you even need this? Send all the updated data to be displayed on the posts through websockets. Read the existing modlog data in reader.go. 

>Also, whatever I do, I do it in that function it currently is in, right?

What?

Reply:
I do it because other clients don't update when I do an a meido vision action.
I send to the server that I just used meido vision, and then server updates all the other clients through the feed. What I'm doing right now should be the same as client/Post.send

Reply:
Why not just do that in the db module? No extra HTTP cal needed.

Reply:
So I need to put that meido vision was used in the db?
Would the db then send immediately that meidovision was used to the server along with whatever else?